tests/utils.py

```diff
@@ -50,20 +51,23 @@ def assert_package_consistency(package_path):
     Return nothing, but raise RuntimeError if inconsistencies are found.
     """
     import tarfile
+
     try:
         with tarfile.open(package_path) as t:
             # Read info from tar file
             member_list = t.getnames()
-            files = t.extractfile('info/files').read().decode('utf-8')
+            files = t.extractfile("info/files").read().decode("utf-8")
             # Read info/has_prefix if present
-            if 'info/has_prefix' in member_list:
+            if "info/has_prefix" in member_list:
```

Comment:
is this filename fixed?? If so this can be an constant itself. (Following a pattern idea

Reply:
this is just unchanged test code that was reformatted